lstchain/reco/r0_to_dl1.py

```diff
@@ -381,18 +416,12 @@ def r0_to_dl1(
 
                 dl1_container.fill_event_info(event)
 
-                tel = event.dl1.tel[telescope_id]
-                tel.prefix = ''  # don't really need one
-                # remove the first part of the tel_name which is the type 'LST', 'MST' or 'SST'
-                tel_name = str(subarray.tel[telescope_id])[4:]
 
                 if custom_calibration:
                     lst_calibration(event, telescope_id)
 
-                write_event = True
```

Comment:
what was this `write_event` doing and why is it now removed?

Reply:
It was unused.